Sources/ApolloCodegenLib/Frontend/JavaScript/src/__tests__/deprecatedEnumCasesCompilationTests.ts

```diff
@@ -0,0 +1,64 @@
+import { 
```

Comment:
Awesome job on this! We don't have many tests for the JS, nice to be adding more.

Worth calling out: I don't think we should spend too much time on writing tests here b/c ideally we move over to use `apollo-rs` at some point in the future! Easy additive tests like this are great, but I wouldn't take the time to go back and write comprehensive tests for the whole thing until we move over to `apollo-rs` (if that ever happens).